core/services/ocr2/plugins/ocr2keeper/evm21/block_subscriber.go

```diff
@@ -141,26 +117,40 @@ func (hw *BlockSubscriber) Start(_ context.Context) error {
 	return hw.sync.StartOnce("BlockSubscriber", func() error {
 		hw.mu.Lock()
 		defer hw.mu.Unlock()
-		_, hw.unsubscribe = hw.hb.Subscribe(&headWrapper{headC: hw.headC})
 		hw.ctx, hw.cancel = context.WithCancel(context.Background())
```

Comment:
use the context argument (currently ignored) instead of `context.Background`

Reply:
done